static/js/public/snap-details/map.js

```diff
@@ -1,29 +1,33 @@
-/* global d3, topojson */
-
 import SnapEvents from '../../libs/events';
+import { select as d3Select, mouse as d3Mouse } from 'd3-selection';
```

Comment:
Not sure if we need `d3...` prefixing of the names. In scope of the module it will probably be clear enough what given `select`, `json`, `geoPath`, etc is. Or it will be easy enough to check where it's imported from.